presto-main/src/test/java/com/facebook/presto/operator/aggregation/reservoirsample/TestUnweightedDoubleReservoirSample.java

```diff
@@ -58,17 +58,17 @@ public void testMany()
 
         long streamLength = 1_000_000;
         for (int i = 0; i < streamLength; ++i) {
-            double value = i * 10 + i % 2;
-            reservoir.add(value);
+            reservoir.add(i);
         }
 
         double[] quantized = new double[4];
         for (double sample : reservoir.getSamples()) {
-            int index = (int) (4.0 * sample / (10.0 * streamLength + 1));
+            int index = (int) (4.0 * sample / (streamLength + 1));
             ++quantized[index];
         }
+        int expectedMin = 25;
```

Comment:
so the number is changed from 35 to 25, assume this is intentional to make sure test is no longer that flaky? ;)